### PR TITLE
translatelocally: init at unstable-2023-08-25

### LINKED ISSUE #250867: Packaging request: translateLocally
**Project description**

> Fast and secure translation on your local machine with a GUI,
> powered by marian and Bergamot.

The program includes both a CLI and GUI.
(The GUI could probably be made optional, or split into two packages).

Language model files need to be downloaded to use the application.
Those could be packaged as Nix packages as well.

**Metadata**

* homepage URL: <https://translatelocally.com/>
* source URL: <https://github.com/XapaJIaMnu/translateLocally>
* license: mit
* platforms: unix, linux, darwin

### PR DESCRIPTION
**Pull request by pacien**
Upstream doesn't have proper version tags, so I'm using a commit hash
and date here. The CMake files are patched so they don't try using
the stripped .git repo data to generate a version string.

Github: closes #250867

### CODE REVIEW

**Review by minijackson — APPROVED** [review]
Looks good :+1:

**Comment by Krutonium** [comment]
LGTM

**Review by Krutonium — APPROVED** [review]

File: pkgs/applications/misc/translateLocally/default.nix [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{ lib, stdenv, fetchFromGitHub
+, cmake, qt6, libarchive, pcre2, protobuf, gperftools, blas
+}:
+
+let
+  rev = "f8a2dba0a63989c6b3a7be36f736ed478cad1dd2";
+
+in stdenv.mkDerivation (finalAttrs: {
+  pname = "translateLocally";
```
Comment (wegank) [inline pkgs/applications/misc/translateLocally/default.nix:9]:
```suggestion
  pname = "translatelocally";
```

Comment (pacien) [reply pkgs/applications/misc/translateLocally/default.nix]: Renamed and amended+squashed.

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -13932,6 +13932,8 @@ with pkgs;
 
   transifex-cli = callPackage ../applications/misc/transifex-cli { };
 
+  translateLocally = callPackage ../applications/misc/translateLocally { };
```
[the remark this answers is not in the document]
Comment (pacien) [reply pkgs/top-level/all-packages.nix]: Renamed and amended+squashed.

**Comment by wegank** [comment]
Hmmm... Mark as broken on aarch64-linux?

**Comment by pacien** [comment]
Quoting Weijia Wang (2023-09-08 22:20:13)
> Hmmm... Mark as broken on aarch64-linux?

Done and amended.

**Pull request merged**